src/uucore/src/lib/features/utmpx.rs

```diff
@@ -191,7 +191,7 @@ impl Utmpx {
     /// A.K.A. ut.ut_tv
     pub fn login_time(&self) -> Tm {
         time::at(Timespec::new(
-            self.inner.ut_tv.tv_sec as i64,
+            self.inner.ut_tv.tv_sec.into(),
```

Comment:
You did trigger a lint lint that actually also applies to the original code

Reply:
what do you mean?

Reply:
Clippy is saying that it's a useless conversion on Mac.